registry.yaml

```diff
@@ -21527,6 +21527,30 @@ packages:
         asset: knest
         supported_envs:
           - linux/amd64
+  - name: snaplet
+    type: http
+    description: Seed your PostgreSQL database with production-like data so you can code, debug, and test with ease.
+    link: https://www.snaplet.dev/
+    url: https://snaplet-public.s3-accelerate.amazonaws.com/cli/beta/snaplet-{{.OS}}-{{.Arch}}-{{.Version}}
+    overrides:
+      - goos: windows
+        url: https://snaplet-public.s3-accelerate.amazonaws.com/cli/beta/snaplet-{{.OS}}-{{.Arch}}-{{.Version}}.exe
```

Comment:
```suggestion
```

This override isn't needed because aqua completes the file extension `.exe` by default on Windows.